docker-utils/pom.xml

```diff
@@ -22,16 +22,32 @@
 
     <parent>
         <groupId>io.confluent</groupId>
-        <artifactId>common-docker</artifactId>
+        <artifactId>common</artifactId>
```

Comment:
this is definitely not the common pattern. i know this gets us consistent with the previous version, but is there any reason not to get this indirectly by keeping `common-docker` as the parent?

Reply:
yeah, that was kind of left over from testing. I will change it back before merging, thanks for catching that.